test/fixtures/node.yml

```diff
@@ -6,7 +6,6 @@ one:
   status: 1
   type: "note"
   uid: 2
-  nid: 1
```

Comment:
Giving explicit value to a primary key doesn't allow to associate it with other model. Auto generating it allow association with other model.